src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/ProjectExtensions.cs

```diff
@@ -0,0 +1,25 @@
+using Xamarin.Android.Tasks;
+using Xamarin.ProjectTools;
+
+namespace Xamarin.Android.Build.Tests;
+
+public static class ProjectExtensions
+{
+	/// <summary>
+	/// Sets the appropriate MSBuild property to use a specific .NET runtime.
+	/// </summary>
+	public static void SetRuntime (this XamarinProject project, AndroidRuntime runtime)
```

Comment:
Could add this here if it needs to be centralized:

```
if (runtime == AndroidRuntime.CoreCLR || runtime == AndroidRuntime.NativeAOT) {
	proj.SetProperty ("EnablePreviewFeatures", "true");
}
```

Reply:
Yes, that seems like a good idea. 👍